cmd/antrea-agent/options.go

```diff
@@ -306,6 +306,23 @@ func (o *Options) validateAntreaIPAMConfig() error {
 	return nil
 }
 
+func (o *Options) validateMulticlusterConfig(encapMode config.TrafficEncapModeType) error {
+	if !o.config.Multicluster.Enable {
+		o.config.Multicluster.EnableStretchedNetworkPolicy = false
+		return nil
+	}
+
+	if !features.DefaultFeatureGate.Enabled(features.Multicluster) {
+		return fmt.Errorf("Multicluster can only be enabled when Multicluster featureGate is enabled")
+	}
+
+	if encapMode != config.TrafficEncapModeEncap {
+		// Only Encap mode is supported for Multi-cluster feature.
```

Comment:
Please update the comment and log message too.

Reply:
Is this comment left before this one https://github.com/antrea-io/antrea/pull/3914#discussion_r1043723189 ?

Reply:
Yes, but maybe you can make it clearer by changing "Multi-cluster feature" to "Multi-cluster Gateway".

Reply:
Done.

Reply:
Done.